device/ethernet.yaml

```diff
@@ -4,13 +4,13 @@ components:
       x-include: 
       - '../common/common.yaml#/components/schemas/Named.Object'
       description: >-
-        Base ethernet interface
+        Base Ethernet interface.
       type: object
       required: [mac]
       properties:
         mac:
           description: >-
-            Media access control address
+            Media Access Control address
           type: string
           format: mac
         mtu:
```

Comment:
This is old but just notifying. When expanding abbreviations, each word should be capitalized. e.g. MTU -> `Maximum Transmission Unit` instead of `Maximum transmission unit`

Reply:
Maximum Transmission Unit.